research/conditional/utils/conditional_trainer.py

```diff
@@ -384,7 +421,7 @@ def _log_train_stats(self, loss_value, step):
                     title=name,
                     value=stats.acc / stats.interval,
                     iteration=step,
-                    processed_token_scale=True,
+                    processed_tokens_so_far=self.num_processed_tokens,
```

Comment:
Since we cannot calculate processed_tokens based on iteration,I think we should change code this way:
```suggestion
                self.logger.report_scalar(
                    title=name,
                    value=stats.acc / stats.interval,
                    iteration=step,
                 )
                 self.logger.report_scalar(
                    title=name,
                    value=stats.acc / stats.interval,
                    iteration= processed_tokens_so_far,
                 )
            
```